src/RustyObjectStore.jl

```diff
@@ -45,21 +45,34 @@ $TYPEDFIELDS
     cache_ttl_secs::Culonglong
     "The time-to-idle in seconds for entries in the client cache"
     cache_tti_secs::Culonglong
+    "Put requests with a size in bytes greater than this will use multipart operations"
+    multipart_put_threshold::Culonglong
```

Comment:
If I'm reading this right, there is a 5MiB minimum on multipart put size: https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html  

If that is true, then I imagine setting `multipart_put_threshold` below 5MiB would either fail for files larger than 5MiB or use a regular put if the file is smaller. Would be good to test this and maybe validate user input if <5MiB doesn't make much sense.

This is also relevant for Nicks question about `multipart_put_part_size` equivalent.

Reply:
It shouldn't fail, it will just be a multipart operation with a single part. The last part is allowed to be smaller than 5MB. I tested this locally with minio and it worked. It's not easy to add a test for this since we can only set the config once.